testing/haxe/APKBUILD

```diff
@@ -24,8 +26,8 @@ build() {
 	cd "$_builddir"
 
 	rmdir libs extra/haxelib_src || return 1
-	mv "$srcdir"/haxelib-* extra/haxelib_src || return 1
-	mv "$srcdir"/ocamllibs-* libs || return 1
+	mv "$srcdir"/haxelib-${_commit_haxelib} extra/haxelib_src || return 1
+	mv "$srcdir"/ocamllibs-${_commit_ocamllibs} libs || return 1
```

Comment:
We can remove the `|| return 1`. abuild will error on any failing command nowdays since we changed to `set -e`.